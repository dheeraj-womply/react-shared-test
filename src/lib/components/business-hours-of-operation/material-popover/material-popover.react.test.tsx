import React from 'react';
import renderer from 'react-test-renderer';
import { render, fireEvent } from '@testing-library/react';
import { MaterialPopover } from '.';

test('Test snapshot for Material Popver component when mounting element is not provided', () => {
    const onClose = jest.fn();
    const component = renderer.create(
        <MaterialPopover 
            element={null}
            onClose={onClose}
            />
    );
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
});
