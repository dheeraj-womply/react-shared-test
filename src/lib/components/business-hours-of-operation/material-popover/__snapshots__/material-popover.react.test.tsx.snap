// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test snapshot for Material Popver component when mounting element is not provided 1`] = `null`;
